swap_meet/item.py

```diff
@@ -1,2 +1,34 @@
+import math
+
 class Item:
-    pass
\ No newline at end of file
+    
+    # age is optional parameter to not affect original tests
+    def __init__(self, category="", condition=0, age=None):
+        self.category = category
+        self.condition = condition
+        # age will be an integer to represent how long (yrs) vendor has owned it
+        self.age = age
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
Great job inserting this method here so all the child classes can also access the descriptions!